saltyrtc/client.ts

```diff
@@ -304,7 +307,14 @@ export class Client {
 
         // Resend offer if signaling channel stays open and peer connection was reset
         if (peerConnection && !signaling && !this.keyStore.hasOtherKey()) {
-            this.peerConnection.sendOffer();
+            this.peerConnection.createOffer().then(
+                (offer) => {
+                    this.peerConnection.setLocalDescription(offer); // TODO: Should we handle errors?
+                    this._startConnectTimer();
+                    this.signaling.sendOffer(offer);
+                },
+                (error) => console.error('PeerConnection error:', error)
+            );
```

Comment:
OTOH if the user manages the peerconnection independently from the SaltyRTC client (like in the high level call flow), then we can remove it from the client and it doesn't matter :)